make/migrations/postgresql/0005_1.9.0_schema.up.sql

```diff
@@ -0,0 +1,29 @@
+/*add webhook policy table*/
+create table webhook_policy (
+ id SERIAL NOT NULL,
+ name varchar(256),
+ project_id int NOT NULL,
+ enabled boolean NOT NULL DEFAULT true,
+ description text,
+ targets text,
+ hook_types text,
+ creator varchar(256),
+ creation_time timestamp default CURRENT_TIMESTAMP,
+ update_time timestamp default CURRENT_TIMESTAMP,
+ PRIMARY KEY (id)
+ );
+
+/*add webhook job table*/
+ CREATE TABLE webhook_job (
+ id SERIAL NOT NULL,
+ policy_id int NOT NULL,
+ status varchar(32),
+ hook_type varchar(256),
+ notify_type varchar(256),
```

Comment:
what is notify_type? can you add some comments for these attributes?

Reply:
> what is notify_type? can you add some comments for these attributes?

notify_type is the hook send method， like HTTP or Email。 I will add comments in schema

Reply:
Is Email a webhook?

Reply:
> Is Email a webhook?

yes, will change webhook->notification